client/client.go

```diff
@@ -420,28 +421,18 @@ func (r *NotaryRepository) Publish(getPass passwordRetriever) error {
 	if err != nil {
 		return err
 	}
-
+	update := make(map[string]json.RawMessage)
```

Comment:
For now it's here. I intend to go through notary and gotuf and as much as possible change them out for `[]byte` but that's time consuming and of minimal impact. The `json` package is already imported.